src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -119,9 +119,9 @@ clean {
   delete 'all-jars'
 }
 
-============== file: samples/src/main/java/com/google/example/examples/library/v1/snippets/DiscussBookCallableCallableStreamingBidiProg.java ==============
+============== file: samples/src/main/java/com/google/example/examples/library/v1/DiscussBookCallableCallableStreamingBidiProg.java ==============
```

Comment:
And capturing our offline discussion for future reference to address a previous comment: we're keeping `examples/` in this path because that is what the manual samples currently use. We can change it later separately if we get agreement from current users/maintainers.

Reply:
recorded it in JavaSurfaceNamer